src/docs/product/accounts/sso/index.mdx

```diff
@@ -147,6 +147,16 @@ In your Rippling admin dashboard, locate the Sentry app in the list of suggested
 
 When prompted with the Rippling Metadata URL, copy this into the Sentry Rippling provider configuration. You will have to complete the Rippling application configuration before completing the sentry provider configuration.
 
+#### Jumpcloud
+
+In your Jumploud Admin portal, navigate to User Authentication > SSO. Add a new SSO Application, and search for Sentry. Click "Configure". Under the SSO menu, update the SP Entity ID and ACS URLs for your tenant. They should be in the format as described [here](https://docs.sentry.io/product/accounts/sso/saml2/#register-sentry-with-idp). Click "save".
```

Comment:
I've formatted this as steps to make it easier for the user to follow and not get lost. 
Also, is the "S" in "save" lowercase in the Jumpcloud UI?  

```suggestion
In your Jumploud Admin portal: 

1. Navigate to **User Authentication > SSO**. 
1. Add a new SSO application and search for Sentry. 
1. Click "Configure". 
1. Under the SSO menu, update the SP Entity ID and ACS URLs for your tenant. They should be in the format as described [here](https://docs.sentry.io/product/accounts/sso/saml2/#register-sentry-with-idp). 
1. Click "save".
```

Reply:
yes, it is lowercase.